src/pjm/parser.jl

```diff
@@ -0,0 +1,152 @@
+const DayAheadHourlyLMP_values_keys = [
+    "system_energy_price_da",
+    "total_lmp_da",
+    "congestion_price_da",
+    "marginal_loss_price_da",
+]
+
+const DayAheadHourlyLMP_meta_keys = [
+    "pnode_name",
+    "voltage",
+    "equipment",
+    "type",
+    "zone",
+    "row_is_current",
+    "version_nbr",
+]
+
+struct DayAheadHourlyLMP <: ElectricityMarket
```

Comment:
This can become ambiguous since all markets (PJM, MISO, ...) have Day-Ahead Hourly LMPs.

Reply:
I would suggest to have a market that is either DayAheadMISO or just MISO with functions to fetch both DA and RT.

Reply:
I am not against having a helper struct, but I don't think it is a market. It could be a subtype of something like AbstractDataStruct

https://github.com/LAMPSPUC/ElectricityMarketData.jl/blob/829c1d04ad18e8bea6657010d226bb43f4c26b55/src/markets/miso_market.jl#L7